CONTRIBUTING.rst

```diff
@@ -1,54 +1,51 @@
 
 ============================
-How to submit a Pull Request
+Submitting a Pull Request
 ============================
 
-**When making changes:**
+a. **Forking the repository:**
 
-1. Create a fork of Tensile---please do not create feature branches directly in https://github.com/ROCm/Tensile.
-2. Clone your fork locally and set up your :ref:`development-environment`.
-3. Create a feature branch off of **develop** and make changes to the code.
-4. Issue ``tox run -m precommit`` and ensure that all checks pass.
-5. Commit you changes using the convention for :ref:`commit-messages`.
-6. If you are updating documentation, issue ``tox run -e docs`` and verify the styling and formatting is what you expect.
-7. Push the changes to your fork.
+   1. Create a fork of Tensile. Do not create feature branches directly in https://github.com/ROCm/Tensile.
+   2. Clone your fork locally and set up your :ref:`development-environment`.
+   3. Create your feature branch from **develop** and make changes to the code.
+   4. Issue ``tox run -m precommit`` and ensure that all checks pass.
+   5. Commit your changes using the convention for :ref:`commit-messages`.
+   6. If you are updating documentation, issue ``tox run -e docs`` and verify the styling and formatting.
+   7. Push the changes to your fork.
 
 .. tip::
 
-   Keeping the scope of new PRs as narrow as possible improves the chances it will be accepted. If you are making multiple changes, consider breaking them into separate PRs. Keeping PRs small supports timely code reviews, traceability, and straightforward reversions.
+   Keeping the scope of new PRs as narrow as possible improves the chances of it getting accepted. If you are making multiple changes, consider breaking them into separate PRs. Keeping PRs small supports timely code reviews, traceability, and straightforward reversions.
 
-**When opening a PR:**
+b. **Creating the PR:**
 
-1. Ensure that **your develop** branch is up-to-date with the **upstream develop** branch---this may require a rebase or a merge.
-2. Verify that your changes pass static analysis checks and all pre-checkin, host library, and unit tests by running ``tox run -m prepr``---then go get a coffee, this could take up to an hour.
-3. Create the PRs against the https://github.com/ROCm/Tensile **develop** branch.
-4. Fill in as many details as possible. Include a description, outcomes, notable changes, and environment information. This more information, the more likely the PR will be reviewed and merged in a timely manner.
-5. Title the PR in present imperative tense, e.g., "*Update* kernel parameters" not "Updates" nor "Updated".
+   1. Ensure that **your develop** branch is up-to-date with the **upstream develop** branch. This might require a rebase or a merge.
+   2. Verify that your changes pass static analysis checks and all pre-checkin, host library, and unit tests by running ``tox run -m prepr``.
```

Comment:
@bstefanuk We have provided some related information in programmer's guide as well, but I couldn't find this command there. Is there any overlap that can be avoided?

Reply:
@bstefanuk Please respond to this as well.

Reply:
Same as above, this is fine to leave in the contributor guide.